lib/task.js

```diff
@@ -99,6 +99,23 @@ async function buildVolumeBindings(taskVolumeBindings, volumeCache, taskScopes)
   return [caches, bindings];
 }
 
+function buildDeviceBindings(devices) {
+  let deviceBindings = [];
+  for (let deviceType in devices) {
+    let device = devices[deviceType];
+    device.mountPoints.forEach((mountPoint) => {
+      deviceBindings.push(
```

Comment:
Shame that this couldn't be as straight forward as docker volume bindings like:
<path on host>:<path in container>:<permissions>